### ISSUE TITLE
v1.5.0版本集成到React的巨坑

### ISSUE THREAD

**Issue by Lzw2016** [pandao/editor.md#704]
### CSS字体文件多了一个小数点导致webpack打包报错

![图片](https://user-images.githubusercontent.com/19291563/61140486-1c013380-a4fe-11e9-881c-039db270de46.png)

![图片](https://user-images.githubusercontent.com/19291563/61140601-49e67800-a4fe-11e9-9b21-c69575a2a499.png)

这问题搞得我一个Java程序员，修改webpack配置进4个小小时，以为字体文件加载处理问题

**Comment by jaradc** [comment]
Yep. I think literally anyone that has tried to get this project to work has encountered this error. It's a total shame. This project is abandoned and this huge, gapping mistake was left in the source code that everyone downloads, encounters, wastes hours of time trying to solve, and eventually realizes there are three "..." and there should only be two "..".
